packages/caip/src/adapters/mtpelerin/index.ts

```diff
@@ -20,31 +22,39 @@ import {
 const MtPelerinSymbolToAssetIds: Record<string, AssetId[]> = {
   ETH: [ethAssetId, optimismAssetId],
   MATIC: [polygonAssetId],
+  XDAI: [gnosisAssetId],
   BTC: [btcAssetId],
   DAI: ['eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f'],
   USDT: [
     'eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7',
     'eip155:137/erc20:0xc2132d05d31c914a87c6611c10748aeb04b58e8f',
+    'eip155:100/erc20:0x4ECaBa5870353805a9F068101A40E0f32ed605C6',
   ],
   USDC: [
     'eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
     'eip155:43114/erc20:0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e',
     'eip155:10/erc20:0x7f5c764cbc14f9669b88837ca1490cca17c31607',
     'eip155:137/erc20:0x2791bca1f2de4661ed88a30c99a7a9449aa84174',
+    'eip155:100/erc20:0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83',
   ],
   jEUR: [
     'eip155:1/erc20:0x0f17bc9a994b87b5225cfb6a2cd4d667adb4f20b',
     'eip155:137/erc20:0x4e3decbb3645551b8a19f0ea1678079fcb33fb4c',
+    'eip155:100/erc20:0x9fB1d52596c44603198fB0aee434fac3a679f702',
   ],
   jCHF: [
     'eip155:1/erc20:0x53dfea0a8cc2a2a2e425e1c174bc162999723ea0',
     'eip155:137/erc20:0xbd1463f02f61676d53fd183c2b19282bff93d099',
+    'eip155:100/erc20:0x2d5563da42b06FbBF9c67b7DC073cF6A7842239e',
```

Comment:
jEUR and jCHF are unsupported on our side I reckon?

<img width="1045" alt="image" src="https://github.com/shapeshift/web/assets/17035424/3ed9e385-444d-4229-91b3-e1c33f7bb87f">

Reply:
Confirmed unsupported, resolved as these have been reverted